.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+
+env:
+  go-version: "1.14"
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Setup Go
+      uses: actions/setup-go@v1
+      with:
+        go-version: ${{ env.go-version }}
+    - name: Build
+      run: |
+        chmod +x build.sh
+        ./build.sh
```

Comment:
What about upload output to artifact? Then auto release can ship artifacts uploaded here.